redaxo/src/addons/phpmailer/pages/checkmail.php

```diff
@@ -17,6 +17,7 @@
     $content .= '</div>';
 } else {
     $mail = new rex_mailer();
+    $mail->FromName = 'REDAXO PHPMailer';
```

Comment:
der absender ist ja auch schon im config formular mit drinn... hier überschreibst du den wert aus dem config formular... d.h. die testmail ist nicht mehr 1:1 wie eine echte mail aus dem system.

kannst du in deiner client software nicht bereits auf das subject `PHPMailer-Test |` filtern (siehe 2 zeilen weiter unten)?

Reply:
Der FromName in der Config ist ja allgemein und wird meist automatisch auch für Antwort-Mails verwendet. Aber du hast Recht, der Betreff sollte für ein Filtern reichen.